core/synch.c

```diff
@@ -1749,6 +1749,14 @@ translate_from_synchall_to_dispatch(thread_record_t *tr, thread_synch_state_t sy
                 arch_mcontext_reset_stolen_reg(dcontext, mc);
             }
         });
+        IF_AARCHXX({
+            // XXX i#4495: Consider saving stolen reg's application value.
```

Comment:
Include the key word "translated".  Otherwise the reader will find this confusing.

Reply:
Done.